node/core/av-store/src/lib.rs

```diff
@@ -1066,6 +1131,78 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			prune_povs: prometheus::register(
+				prometheus::Histogram::with_opts(
+					prometheus::HistogramOpts::new(
+						"parachain_av_store_prune_povs",
+						"Time spent within `av_store::prune_povs`",
+					)
+				)?,
+				registry,
+			)?,
+			prune_chunks: prometheus::register(
```

Comment:
Should consider defining a colusre `|name: &str , desc: &str| -> {}` that does all the setup for a histogram.

Reply:
I'm a bit reluctant to do that because it makes it harder to customize the individual options in the event that we want e.g. custom buckets on a particular histogram.

Reply:
Fair point, if that is something we want to do down the road. Just looking at the changes, there are a lot of repetitions (which makes sense conceptually).